cmd/cluster/node/list_nodes.go

```diff
@@ -63,6 +63,4 @@ var listNodeCmd = &cobra.Command{
 
 func init() {
 	NodeCmd.AddCommand(listNodeCmd)
-	listNodeCmd.Flags().String("cluster-name", "", "[REQUIRED] The name of the cluster to get details.")
```

Comment:
Why removing from here ? I think is best for the command options to be within the same file where the command / subcommand are defined.

Reply:
My thought was that all commands under `cluster` would need to specify the `cluster-name`, so instead of having this mandatory parameter mentioned everywhere, we would have it in a centralized place. If you feel strongly, I can undo this.